docs/jobs/index.md

```diff
@@ -39,7 +39,7 @@ We believe that **decentralized organizations can solve the world's worst proble
 ## Openings 🙋🏾🙋‍♂️
 
 * [EVM/Solidity Engineer](openings/solidity.md)
-* [Frontend and web3 Developer](openings/frontend.md)
+* [Frontend/Web3 Developer](openings/frontend.md)
```

Comment:
Small thing: while "engineer" and "developer" are highly synonymous terms in dev land, usually "engineer" holds a slightly higher connotation.

I only note it because our EVM position is listed as an "engineer" while all the others are "developers". We should probably stay consistent between the two.

Reply:
I will leave these decisions to @luisivan and @izqui